dev/doc/changes.md

```diff
@@ -42,11 +42,11 @@ Proof state:
 
 ### ML4 Pre Processing
 
-- Support for `.ml4` files, processed by camlp5 has been removed in
+- Support for `.mlg` files, processed by camlp5 has been removed in
```

Comment:
changes.md should not be changed.  This sentence in particular becomes nonsensical with the change